options.go

```diff
@@ -56,6 +56,7 @@ type Options struct {
 	GitHubTeam               string   `flag:"github-team" cfg:"github_team" env:"OAUTH2_PROXY_GITHUB_TEAM"`
 	GitHubRepo               string   `flag:"github-repo" cfg:"github_repo" env:"OAUTH2_PROXY_GITHUB_REPO"`
 	GitHubToken              string   `flag:"github-token" cfg:"github_token" env:"OAUTH2_PROXY_GITHUB_TOKEN"`
+	GitHubUsers              []string `flag:"github-users" cfg:"github_users" env:"OAUTH2_PROXY_GITHUB_USERS"`
```

Comment:
Sorry, I wasn't very clear in my previous comment. The flag should be singular, while the other two should be pluralised (yes this is very confusing, but it's what we have for now).
```suggestion
	GitHubUsers              []string `flag:"github-user" cfg:"github_users" env:"OAUTH2_PROXY_GITHUB_USERS"`
```

Reply:
Thanks for your explanation, I understood!
https://github.com/oauth2-proxy/oauth2-proxy/pull/421/commits/5dc11924b63c590f3b61f79ff0bac3a15107a138